.github/workflows/close-stale-issues.yml

```diff
@@ -1,30 +1,51 @@
-name: "Close stale Issues"
+name: "Close Stale Issues"
 
+# Controls when the action will run.
 on:
+  workflow_dispatch:
   schedule:
-  - cron: "0 6 * * *"
+  - cron: "0 */4 * * *"
 
 jobs:
   cleanup:
+    permissions:
+      issues: write
+      contents: read
+      pull-requests: write
     runs-on: ubuntu-latest
     name: Stale issue job
     steps:
-    - uses: aws-actions/stale-issue-cleanup@v3
+    - uses: aws-actions/stale-issue-cleanup@v5
       with:
-        ancient-issue-message: This issue has not received any attention in 1 year.  If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: This issue has not received any attention in 1 year. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
         stale-issue-message: This issue has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
         stale-pr-message: This PR has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
-
+        
+        # These labels are required
         stale-issue-label: closing-soon
-        exempt-issue-labels: no-autoclose,feature-request

```

Comment:
Why did we remove `feature-request` from exemption list?

Reply:
Because I copy + pasted this from CDK lol